app/presenters/concerns/hyku_addons/work_presenter_behavior.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
```

Comment:
This concern takes the place of the GenericWorkPresenter, which is more specially targeted to adding functionality to that Presenter, rather than being a generic concern for all work presenters.